### PR TITLE
Restart Relativity

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Very basic script to restart the Relativity Data Server remotely
without needing Windows.

### CODE REVIEW

**Review by cracell — APPROVED** [review]
LGTM 👍 

Though I'm not very familiar with the tools being used.

File: bin/restart [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift File.expand_path('../../', __FILE__)
+
+require 'config/initializers/figaro'
+require 'winrm'
+
+options = {
+  endpoint: format(
+    'http://%s:%s/wsman',
```
Comment (upsidedownchicken) [inline bin/restart]: Dumb question probably but no https available?

**Comment by upsidedownchicken** [comment]
👍

File: bin/restart (hunk 1 above)
Comment (josephbridgwaterrowe) [reply bin/restart]: I didn't try, I will give it a go... worse case I can add to env for those configurations that are using `https` as I believe it should be supported, not sure if our servers are set up for it though.

Comment (josephbridgwaterrowe) [reply bin/restart]: I tried using SSL and received the following error:

```
/Users/josephbridgwaterrowe/.rvm/gems/ruby-2.2.2/gems/httpclient-2.8.3/lib/httpclient/ssl_socket.rb:103:in `connect': SSL_connect returned=1 errno=0 state=SSLv2/v3 read server hello A: unknown protocol (OpenSSL::SSL::SSLError)
```

I'm going to see if we can get SSL enabled on it... to support this in future I will add a setting for protocol.

**Pull request merged**